docs/openapi/components/schemas/common/SC237AForm.yml

```diff
@@ -0,0 +1,189 @@
+$linkedData:
+  term: SC237AForm
+  '@id': https://w3id.org/traceability#SC237AForm
+title: SC-237A Form
+description: Information regarding an application for audit services.
```

Comment:
Can we link to a hosted version of this form from here? We're doing similar for other forms.

Reply:
@mkhraisha 

Do you mean the pdf? As in https://www.ams.usda.gov/sites/default/files/media/SC237A.pdf ?

I'm concerned that a link like that might break, but a link to the USDA page for that form (https://www.ams.usda.gov/resources/sc237a) should work, as it references the form and is more likely to be updated.

I'm open to doing that, but would it work instead to just have a more detailed description (without breakable links)? I've been meaning to do that anyway.

Reply:
yes a detailed description instead works.